packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx

```diff
@@ -132,7 +132,6 @@ export class Dialog extends BaseComponent<IDialogProps, IDialogState> {
                       className='ms-Dialog-button ms-Dialog-button--close'
                       buttonType={ ButtonType.icon }
                       icon='Cancel'
-                      rootProps={ { title: closeButtonAriaLabel } }
```

Comment:
Will this cause errors for anyone who's currently using `rootProps`? I don't see this prop in our documentation and am not clear on how long it was deprecated for.